From ad6969f267cdb30a14b8189a19d88adde02a812c Mon Sep 17 00:00:00 2001
From: Adrien Ricciardi <aricciardi@baylibre.com>
Date: Tue, 7 Nov 2023 17:08:57 +0100
Subject: [PATCH 3/3] examples: spider: Used the CAN controller 1 in the CAN
 demo application.

Also removed the now unused serial library from the build.

Signed-off-by: Adrien Ricciardi <aricciardi@baylibre.com>
---
 examples/cortex-a/armv8/spider/can_demo/can_demo.c   | 4 ++--
 examples/cortex-a/armv8/spider/can_demo/can_demo.oil | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/examples/cortex-a/armv8/spider/can_demo/can_demo.c b/examples/cortex-a/armv8/spider/can_demo/can_demo.c
index bd96e182..53e51905 100644
--- a/examples/cortex-a/armv8/spider/can_demo/can_demo.c
+++ b/examples/cortex-a/armv8/spider/can_demo/can_demo.c
@@ -36,9 +36,9 @@ int main(void)
 	// Statically list the configuration of each CAN controller used in the application
 	static tpl_can_controller_config_t can_controllers_config[] =
 	{
-		// First controller will use CAN 2.0
+		// Use the second CAN controller
 		{
-			&spider_can_controller_0,
+			&spider_can_controller_1,
 			// CAN 2.0 at 250Kbps
 			#if 0
 			{
diff --git a/examples/cortex-a/armv8/spider/can_demo/can_demo.oil b/examples/cortex-a/armv8/spider/can_demo/can_demo.oil
index 0e4956f1..7d0c7099 100644
--- a/examples/cortex-a/armv8/spider/can_demo/can_demo.oil
+++ b/examples/cortex-a/armv8/spider/can_demo/can_demo.oil
@@ -27,7 +27,6 @@ CPU can_task {
 			LINKER = "arm-none-eabi-ld";
 			COPIER = "arm-none-eabi-objcopy";
 			SYSTEM = PYTHON;
-			LIBRARY = serial; // TODO remove after tests
 			LIBRARY = can;
 		};
 
-- 
2.42.0

